src/openms/source/METADATA/ID/IdentificationDataConverter.cpp

```diff
@@ -1214,4 +1214,169 @@ namespace OpenMS
     }
   }
 
+
+  void IdentificationDataConverter::importConsensusIDs(ConsensusMap& consensus,
```

Comment:
what does this function do? can you add a one line comment. seems it copies some stuff between data structures.

Reply:
Okay.

Reply:
Okay.